packages/avatar/src/vaadin-avatar.js

```diff
@@ -66,16 +65,7 @@ class Avatar extends AvatarMixin(ElementMixin(ThemableMixin(PolylitMixin(LitElem
         @error="${this.__onImageLoadError}"
         draggable="false"
       />
-      <svg
-        part="icon"
-        ?hidden="${!this.__iconVisible}"
-        id="avatar-icon"
-        viewBox="-50 -50 100 100"
-        preserveAspectRatio="xMidYMid meet"
-        aria-hidden="true"
-      >
-        <text dy=".35em" text-anchor="middle">&#xea01;</text>
-      </svg>
+      <div part="icon" ?hidden="${!this.__iconVisible}" id="avatar-icon" aria-hidden="true"></div>
```

Comment:
We should also update the snapshot as `<svg>` is not rendered there due to https://github.com/open-wc/open-wc/issues/1229.